config/locales/en.yml

```diff
@@ -310,6 +310,8 @@ en:
         wiki: "Wiki"
       from_other: "From other"
       community_scientists: "community scientists"
+      recent_activity: "Recent activity"
+       post_date: "Post date"
```

Comment:
Great! This is in the right location, you only need to fix one thing here, there's an extra space:

```
      recent_activity: "Recent activity"
      post_date: "Post date"
```

instead of:
```
      recent_activity: "Recent activity"
       post_date: "Post date"
```